### PR TITLE
[291] Lint identify test helpers

### PR DESCRIPTION
**Pull request by pbains**
## Scope

Lint to identify helper tests 

## Why?

Additional lints and https://github.com/kulti/thelper

### CODE REVIEW

File: crypto/bn256/cloudflare/gfp_test.go [hunk 1]
```diff
@@ -706,13 +724,14 @@ func TestIsEqualGFP(t *testing.T) {
...
 	for _, k := range []int{-1, 0, 1} {
-		gfpMarshalTest(k, t)
+		gfpMarshalTest(t, k)
 	}
 	gfpMarshalTestExceed(t)
 	gfpMarshalTestEqual(t)
 }
 
-func gfpMarshalTest(k int, t *testing.T) {
+func gfpMarshalTest(t *testing.T, k int) {
```
Comment (diligentis) [inline crypto/bn256/cloudflare/gfp_test.go:733]: Not sure if this was the linter or you, but nice cleanup.

File: transport/transportFail_test.go [hunk 2]
```diff
@@ -59,7 +61,8 @@ func dialer2(t *testing.T, transport interfaces.P2PTransport, addr interfaces.No
 	}
 }
 
-func accept2(transport interfaces.P2PTransport, complete chan struct{}) {
+func accept2(t *testing.T, transport interfaces.P2PTransport, complete chan struct{}) {
```
Comment (diligentis) [inline transport/transportFail_test.go:64]: `*testing.T` not used.

**Review by diligentis — APPROVED** [review]
One cleanup nit, otherwise good!

**Comment by pbains** [comment]
I need to fix a few more tests.

**Review by diligentis — APPROVED** [review]
LGTM!

**Pull request merged**